### PR TITLE
Fix Issue 17501 - Runnable unittest problem with AST rewrite

### PR DESCRIPTION
**Pull request by wilzbach**
@CyberShadow are you okay with enabling Travis here, s.t. we can easily run `test` on a CI?

It works on my fork:
https://travis-ci.org/wilzbach/dlang.org

image

(I also wanted to revive the PR in which we build dlang.org from scratch on Travis, but that would be another step).

### CODE REVIEW

**Comment by wilzbach** [comment]
CC @ZombineDev  - thanks a lot for reporting!

**Comment by CyberShadow** [comment]
> @CyberShadow are you okay with enabling Travis here, s.t. we can easily run test on a CI?

Yeah but it could use DAutoTest just as well.

**Comment by wilzbach** [comment]
> Yeah but it could use DAutoTest just as well.

I know, but it's a free CI & we get a nice UX separation between failed tests and failed documentation build.
Also [I wanted to use Travis to build dlang.org from scratch to ensure consistency](https://github.com/dlang/dlang.org/pull/1651) in the near future.

**Comment by CyberShadow** [comment]
> I know, but it's a free CI & we get a nice UX separation between failed tests and failed documentation build.

Is this going to eat into Travis time for the other repositories, though?

**Comment by wilzbach** [comment]
> Is this going to eat into Travis time for the other repositories, though?

Yes, but it currently takes less than one minute to run, so imho marginal. 
If I think about it, it doesn't make sense to test it with different versions as we download and use `STABLE_DMD` always anyhow (will make a short-follow, sorry).

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+language: d
+os:
+ - linux
+d:
+ - dmd
+ - dmd-beta
+ - dmd-nightly
+ - ldc
+ - ldc-beta
```
Comment (CyberShadow) [inline .travis.yml]: I think this is overkill. Just test with the DMD built by the makefile.

**Comment by wilzbach** [comment]
>> If I think about it, it doesn't make sense to test it with different versions as we download and use STABLE_DMD always anyhow (will make a short-follow, sorry).
> I think this is overkill. Just test with the DMD built by the makefile.

-> https://github.com/dlang/dlang.org/pull/1695

**Pull request merged**